src/Pages/namjunpark/Main/Components/Feed.js

```diff
@@ -0,0 +1,230 @@
+import React, { Component } from "react";
+import Story from "./Story";
+
+class Feed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: 0,
+      comment: "",
+      comments: [],
+      storyData: [
+        {
+          id: 0,
+          user: "namjunpark",
+          src: "/images/namjunpark/storyImg/12.jpg",
+        },
+        {
+          id: 1,
+          user: "wontaezia",
+          src: "/images/namjunpark/storyImg/13.jpg",
+        },
+        {
+          id: 2,
+          user: "jo0000n",
+          src: "/images/namjunpark/storyImg/14.jpg",
+        },
+        {
+          id: 3,
+          user: "chynma",
+          src: "/images/namjunpark/storyImg/15.jpg",
+        },
+        {
+          id: 4,
+          user: "cholalkw",
+          src: "/images/namjunpark/storyImg/16.jpg",
+        },
+        {
+          id: 5,
+          user: "nasd2",
+          src: "/images/namjunpark/storyImg/17.jpg",
+        },
+        {
+          id: 6,
+          user: "heize",
+          src: "/images/namjunpark/storyImg/18.jpg",
+        },
+        {
+          id: 7,
+          user: "won__i",
+          src: "/images/namjunpark/storyImg/19.jpg",
+        },
+        {
+          id: 8,
+          user: "zdkksb",
+          src: "/images/namjunpark/storyImg/20.jpg",
+        },
+        {
+          id: 9,
+          user: "asdkjna",
+          src: "/images/namjunpark/storyImg/21.jpg",
+        },
+        {
+          id: 10,
+          user: "faall_ina!",
+          src: "/images/namjunpark/storyImg/22.jpg",
+        },
+        {
+          id: 11,
+          user: "nasdljbasdl",
+          src: "/images/namjunpark/storyImg/23.jpg",
+        },
+        {
+          id: 12,
+          user: "nty",
+          src: "/images/namjunpark/storyImg/24.jpg",
+        },
+        {
+          id: 13,
+          user: "mtv",
+          src: "/images/namjunpark/storyImg/25.jpg",
+        },
+        {
+          id: 14,
+          user: "nappo",
+          src: "/images/namjunpark/storyImg/26.jpg",
+        },
+      ],
+    };
+  }
+
+  handleComment = (e) => {
+    this.setState({ id: this.state.comments.length, comment: e.target.value });
+  };
+
+  handleClick = (e) => {
+    if (this.state.comment.length > 0) {
+      this.setState({
+        comments: this.state.comments.concat({
+          id: this.state.id,
+          comment: this.state.comment,
+          like: false,
+        }),
```

Comment:
concat은 배열과 배열을 합치는 메서드라서
아래와 같이 배열로 감싸서 넣어주시는게 정확합니다!
```suggestion
        comments: this.state.comments.concat([{
          id: this.state.id,
          comment: this.state.comment,
          like: false,
        }]),
```